crates/storage/db-common/src/init.rs

```diff
@@ -124,6 +124,12 @@ pub fn init_genesis<DB: Database>(factory: ProviderFactory<DB>) -> Result<B256,
         provider_rw.save_stage_checkpoint(stage, Default::default())?;
     }
 
+    let segment = StaticFileSegment::Receipts;
+    static_file_provider.latest_writer(segment)?.increment_block(segment, 0)?;
+
+    let segment = StaticFileSegment::Transactions;
+    static_file_provider.latest_writer(segment)?.increment_block(segment, 0)?;
```

Comment:
can we document why we're doing this?
I assume this is to account for the genesis block?